docs/docs/advanced/air-gap.md

```diff
@@ -157,6 +105,43 @@ For Java DB the process is the same, except for the following:
 
 ## Misconfigurations scanning
```

Comment:
since the intro says "The rest of this document details each resource's connectivity requirements and relevant configuration options." I'll go with "Misconfiguration Checks database"